x-pack/legacy/plugins/siem/public/routes.tsx

```diff
@@ -12,6 +12,16 @@ import { NotFoundPage } from './pages/404';
 import { HomePage } from './pages/home';
 import { ManageRoutesSpy } from './utils/route/manage_spy_routes';
 
+/* Uncomment only during debugging */
```

Comment:
do we want to keep it like that or adding it to a `read.me`. So it is easy to find it later on.